apps/api/resources.dev/user/cvasquez.ttl

```diff
@@ -1,6 +0,0 @@
-PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
-
-<>
-  a vcard:Individual ;
-  vcard:hasUID "cvasquez" ;
-.
```

Comment:
I removed the dev-env users resources since they will be created from the `creds.json` file